hoomd/pytest/test_simulation.py

```diff
@@ -115,14 +115,21 @@ def test_run(simulation_factory, lattice_snapshot_factory):
     assert sim.operations._scheduled
 
 
-def test_tps(simulation_factory, lattice_snapshot_factory):
+def test_tps(simulation_factory, two_particle_snapshot_factory):
```

Comment:
The fix in this pull request also fixes the `walltime` property. Add a similar test (or include in this one renamed to `test_tps_and_walltime`).